source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,40 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - create
+  - drop
+  - dropDatabase
+  - createIndexes
+  - dropIndexes
+  - collModrefineCollectionShardKey
+  - rename
+  - reshardCollection
+  - shardCollection
+
+  This includes operations that may create new collections such as 
+  :dbcommand:`mapReduce`, :pipeline:`$out`, and :pipeline:`$merge`. This also 
+  includes collections created implicitly from inserts. Only writes that produce 
+  CRUD events can occur during the migration.
+
+  .. note:: 
+   
+     Writes that produce DDL events on source collections outside of the 
+     :ref:`namespace filter <c2c-filtered-sync>` are allowed.
+
+- :ref:`GeoHaystack indexes <index-geohaystack-index>` are not supported.
+
+- :ref:`/reverse <c2c-api-reverse>` endpoints are not supported. You can't 
+  enable ``reversible`` in the :ref:`/start <c2c-api-start>` request.
+
+- You can't enable the ``enableUserWriteBlocking`` in the ``/start`` request. 
+
+- :ref:`SRV connection strings <connections-dns-seedlist>` for 4.4 source 
```

Comment:
Making sure I'm understanding this correctly - this limitation only applies to 4.4 source clusters, not all pre-6.0 clusters?

Reply:
@ajayvijayakumar123 can you confirm on whether this is true?

Reply:
yup just for 4.4 sources

Reply:
Maybe we can reorder the phrases here slightly so it's clearer on a quick scan that this only applies to 4.4. Like:

"For 4.4 source clusters, [SRV connection strings](https://www.mongodb.com/docs/master/reference/connection-string/#std-label-connections-dns-seedlist) are not supported. You must use a [standard connection string.](https://www.mongodb.com/docs/master/reference/connection-string/#std-label-connections-standard-connection-string-format)"

I'd also maybe consider moving this to the end of the list since it's kind of in its own category of only applying to 4.4 clusters.